ross/utils.py

```diff
@@ -0,0 +1,237 @@
+import pandas as pd
```

Comment:
We could evaluate if we really need pandas or if we could do this using only xlrd, since pandas is a big dependency and it also adds some import time.
This is just a general comment for our evaluation in the future since we would need to remove the use of pandas also from the rotor_assembly module.

Reply:
We should open an issue for this one.